regression/scripts/constraint_analysis/constraint_analysis.py

```diff
@@ -44,36 +49,28 @@ def main():
     ca.cruise.altitude        = 35000 * Units['feet']  
     ca.cruise.mach            = 0.78
     ca.cruise.thrust_fraction = 0.85
-    # turn
-    ca.turn.angle           = 15    * Units['degrees']
-    ca.turn.altitude        = 35000 * Units['feet']   
-    ca.turn.mach            = 0.78
-    ca.turn.thrust_fraction = 0.85
-    # ceiling
-    ca.ceiling.altitude    = 36500 * Units['feet'] 
-    ca.ceiling.mach            = 0.78
     # landing
-    ca.landing.ground_roll = 1400 * Units['m']   
+    ca.landing.ground_roll = 1400 * Units['m']  
+
+    # define vehicle-specific properties
+    vehicle  = B737_vehicle()
+
 
     # Default aircraft properties
     # geometry
-    ca.geometry.aspect_ratio           = 9.16
-    ca.geometry.thickness_to_chord     = 0.11
-    ca.geometry.taper                  = 0.3
-    ca.geometry.sweep_quarter_chord    = 25 * Units['degrees']
-    ca.geometry.high_lift_type_clean   = None
-    ca.geometry.high_lift_type_takeoff = 'double-slotted Fowler'
-    ca.geometry.high_lift_type_landing = 'double-slotted Fowler'
+    ca.geometry.aspect_ratio                 = vehicle.wings['main_wing'].aspect_ratio
```

Comment:
I was hoping it would look something like this:

ca.geometry = B737_vehicle()

That way a user doesn't need to do all of this manually. 

I think a really neat workflow for a new user who would be interested in your contributions here could be to draw a vehicle in OpenVSP and very quickly produce the constraint diagram. That would enable this.

Reply:
I am actually working on the neat integration now, so the constraint diagram script will accept the entire vehicle with some pre-defined constraint diagram settings. I can also do it the way you suggested. Hope, I will commit the deep update by the end of next week

Reply:
Great! Do what you think makes most sense here.

I'm looking forward to this. I think your work here fills an important need in SUAVE.